src/features/ccip/ImportCCIPPackage.mdx

```diff
@@ -31,7 +31,7 @@ import { Tabs } from "@components/Tabs"
       If you use [Foundry](https://book.getfoundry.sh/), install the v1.2.1 release:
```

Comment:
```suggestion
      If you use [Foundry](https://book.getfoundry.sh/), install a specific [CCIP GitHub
      release](https://github.com/smartcontractkit/ccip/releases):
```

Reply:
@khadni At the moment, it is preferable to stick to a specific release which is v1.4.0 (just updated the PR)